src/includes/constants.php

```diff
@@ -198,3 +199,18 @@
 if (!defined('TEMP_FOLDER')) {
     define('TEMP_FOLDER', TEMP_PATH);
 }
+
+// Define the Ressource Dir constant that will be use to exposed vendor assets
+if (!defined('RESOURCES_DIR')) {
+    $project = new SilverStripe\Core\Manifest\Module(BASE_PATH, BASE_PATH);
+    $resourceDir = $project->getResourcesDir() ?: 'resources';
```

Comment:
Super minor, but maybe we could call it `$resourcesDir` for consistency

Reply:
Fixed.